src/AppInstallerCLICore/Workflows/UninstallFlow.cpp

```diff
@@ -123,4 +123,70 @@ namespace AppInstaller::CLI::Workflow
 
         context.Reporter.Info() << Resource::String::UninstallFlowUninstallSuccess << std::endl;
     }
+
+    void ReportDependencies(Execution::Context& context)
+    {
+        context <<
+            Workflow::GetManifest <<
+            Workflow::SelectInstaller;
+            
+        const auto& installer = context.Get<Execution::Data::Installer>();
+        if (installer)
+        {
+            auto dependencies = installer->Dependencies;
+            if (dependencies.HasAny())
+            {
+                context.Reporter.Info() << "This package had dependencies that may not be needed anymore:" << std::endl;
+
+                auto windowsFeaturesDep = dependencies.WindowsFeatures;
```

Comment:
Definitely too many copies of this floating around.  A single workflow task should be able to handle the reporting here.  I would keep it as `Report*` as it will need to stay that long term, then the action tasks can call `Report*` for now and eventually be replaced with the actual action.